### ISSUE TITLE
Allow to set how much data is "quite a lot"

### ISSUE THREAD

**Issue by RicoElectrico** [tyrasd/overpass-turbo#248]
For decent PCs 2 or even 10 MB of data on the map is not going to cause serious problems, there may be some lag, but no freezing.

**Comment by tyrasd** [comment]
This is true, but it also depends very much on what kind of data is returned: While 10MB of large, complex polygons is typically no problem at all, but 10MB of points may be an issue (because of the large number of created SVG/DOM elements).

But, I admit that a customizable limit does indeed make sense.

**Comment by DaveF63** [comment]
Are these requests related to #269?

**Comment by tyrasd** [comment]
suggestion from @bjohas: make this configurable on a per-query basis: e.g. something like `{{largeDataWarningLimit:10MB}}`

**Comment by DaveF63** [comment]
Is there a reason this was not implemented? 
In the nine years since it's been raised 2mb has become even more of an unnecessarily low number.

**Comment by HolgerJeromin** [comment]
> Is there a reason this was not implemented?

Because no one gets paid to do stuff in this repo.
And nobody had interest enough to implement for fun.

**Comment by DaveF63** [comment]
Sadly, this appears to be a common attitude with many OSM related software projects. 
OSM database contributors don't get paid, yet do it for fun.

**Comment by DaveF63** [comment]
This implementation is not stable. I'm still getting the occasional message for just 1mb.
  
<img width="875" height="266" alt="Image" src="https://github.com/user-attachments/assets/dd39be97-f5d2-4ba4-8825-bcc0317cbbc7" />

I'm pretty sure my requests aren't hitting the "either more than 5k "features"
*or ~10MB of data" limits.
